cmd/main.go

```diff
@@ -267,6 +270,14 @@ func getOutputFn(outputFlag *string, currentFn selector.InstanceTypesOutputFn) s
 			return selector.InstanceTypesOutputFn(outputs.TableOutputWide)
 		case tableOutput:
 			return selector.InstanceTypesOutputFn(outputs.TableOutputShort)
+		case oneLine:
+			return selector.InstanceTypesOutputFn(func(instanceTypeInfoSlice []*ec2.InstanceTypeInfo) []string {
```

Comment:
We don't want this in outputs.go ? 

Kinda ironic that the oneLine function is the online function in the switch-case that isn't one-line :p

Reply:
fair point, and should probably add  a simple test :D